pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "pinecone-text"
-version = "0.6.1"
+version = "0.7.0"
```

Comment:
@igiloh-pinecone I'm not 100% sure if it should be 0.6.2 or 0.7.0 because it's do a breaking change in terms of dependency, but over a very short living feature and API is the same

Reply:
It should be `0.7`. By definition patch versions are bug fixes only that don't break\change any APIs.  
(Theoretically as long as our major version is `0` we're allowed to break an APIs in any version, according to SemVer rules - but let's try no to abuse that...)